tests/bluetooth/tester/src/btp_bas.c

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/services/bas.h>
+#include "btp/btp.h"
+#include "btp/btp_bas.h"
+
+static uint8_t bas_supported_commands(const void *cmd, uint16_t cmd_len, void *rsp,
+				      uint16_t *rsp_len)
+{
+	struct btp_bas_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_BAS_READ_SUPPORTED_COMMANDS);
+	tester_set_bit(rp->data, BTP_BAS_SET_BATTERY_LEVEL);
+	tester_set_bit(rp->data, BTP_BAS_BLS_SET_BATTERY_PRESENT);
+	tester_set_bit(rp->data, BTP_BAS_BLS_SET_WIRED_POWER_SOURCE);
+
+	/* octet 1 */
```

Comment:
Why are we only using 4 bits in the first octet?

Doesn't `tester_set_bit(rp->data, BTP_BAS_BLS_SET_WIRELESS_POWER_SOURCE);` set it at BIT(5), which is octet 0 and not octet 1?

Reply:
This was a suggestion from @sjanc . 
Perhaps I have misunderstood it

Reply:
Since `tester_set_bit` basically just does `BIT()`, any values below 0x08 ends up in octet 0, values from 0x08->0x0F (inclusive) ends up in octet 1

Reply:
So it's just the comment that should be moved so that it matches the values